wire-ios-data-model/Source/MLS/CoreCryptoKeyProvider.swift

```diff
@@ -87,7 +87,7 @@ struct CoreCryptoKeychainItem: KeychainItemProtocol {
         ]
     }
 
-    func setQuery<T>(value: T) -> [CFString: Any] {
+    func setQuery(value: some Any) -> [CFString: Any] {
```

Comment:
it doesn't make sense, I think it should resolve this to `Any`, not `some Any`

Reply:
I think that is the new generics API if we don't constrain the generic to a specific type.